crates/kitsune_p2p/types/benches/api_thru.rs

```diff
@@ -1,144 +1,148 @@
-#![allow(irrefutable_let_patterns)]
-use criterion::{/*black_box,*/ criterion_group, criterion_main, Criterion};
-
-use futures::stream::StreamExt;
-use kitsune_p2p_types::tx2::tx2_api::*;
-use kitsune_p2p_types::tx2::tx2_pool_promote::*;
-use kitsune_p2p_types::tx2::tx2_utils::*;
-use kitsune_p2p_types::tx2::*;
-use kitsune_p2p_types::*;
-
-const SIZE: usize = 2048;
-
-kitsune_p2p_types::write_codec_enum! {
-    codec TestData {
-        One(0x01) {
-            data.0: PoolBuf,
-        },
-    }
+fn main() {
+    todo!("rewrite for tx5")
```

Comment:
Tracked somewhere? Benchmark tests would be a useful thing to have a bit of focus on at some point in the reasonably near future. Or at least it's something I've been thinking I would like to have a bit more of.

Reply:
@neonphog is this benchmark still useful or is it covered by something else?

Reply:
no, and no. tx5 itself has benchmarks, but the kitsune integration of tx5 doesn't have benchmarks. But I don't think the tx2 benchmark will be useful... we'd just need to rewrite it.

Reply:
I'll keep this as-is for now, as a reminder to rewrite it.